### DISCUSSION TITLE
Animate vs Intellij builds create different bitmap performance behaviors

### DISCUSSION THREAD

**Discussion by toysonlocation** [airsdk/Adobe-Runtime-Support#1981] [General]
Hi all,

Forever I've noticed that bitmaps in my pure AS3 mobile builds for iOS run uglier when I compile from Intellij than from Animate. For example, rotating bitmaps in very small increments is a little jumpy from IntelliJ, and much smoother from Animate. Same codebase, SDK, and manifest. Is there some special config or wrapper that Animate is applying? Please advise if you have any knowledge of this.

My main class has this at the front (and this class is used as the Document class in Animate)
    [SWF(frameRate="60", width="750", height="1334", backgroundColor="#000000")]

Best,
Joseph

**Comment by ajwfrost** [comment]
How are these bitmaps being included in the application? it may be that Animate is encoding them with a slightly reduced quality (using jpeg format) and maybe Intellij is using PNG or a jpeg with 100% quality level?

If you can create two SWFs, one from each of Animate and Intellij, we can inspect these to see what the encodings are (you can use SWF Investigator to see the tag details, plus the SWF format spec sometimes helps..)

thanks
